testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -642,16 +632,38 @@ impl ClusterSwarmKube {
         try_join_all(delete_futures).await?;
         Ok(())
     }
+
+    /// Runs command on the provided host in separate utility container based on cluster-test-util image
+    pub async fn util_cmd<S: AsRef<str>>(
+        &self,
+        command: S,
+        k8s_node: &str,
+        job_name: &str,
+    ) -> Result<()> {
+        self.run(
+            k8s_node,
+            "853397791086.dkr.ecr.us-west-2.amazonaws.com/cluster-test-util:latest",
+            command.as_ref(),
+            job_name,
+        )
+        .await
+    }
 }
 
 #[async_trait]
 impl ClusterSwarm for ClusterSwarmKube {
-    async fn spawn_new_instance(
-        &self,
-        instance_config: InstanceConfig,
-        delete_data: bool,
-    ) -> Result<Instance> {
-        self.upsert_node(instance_config, delete_data).await
+    async fn spawn_new_instance(&self, instance_config: InstanceConfig) -> Result<Instance> {
+        self.upsert_node(instance_config).await
+    }
+
+    async fn clean_data(&self, instance_config: &InstanceConfig) -> Result<()> {
```

Comment:
I think it would be better to change this method to just accept node, instead of instance.
In conjunction with your previous PR this also should be easy to call, since you will have a node in cluster builder

Reply:
Yes, that is what I plan to do once the previous pr is approved.